packages/cli/src/commands/prisma/generate.js

```diff
@@ -1,24 +1,15 @@
+// helper used in Dev and Build commands
```

Comment:
`db generate` had specific logic for use with Build and Dev commands. I converted to a helper and moved to `commands/prisma/generate.js`. Perhaps best to move into `prisma.js`? Suggestions otherwise?